site/app/templates/submission/homework/BulkUploadBox.twig

```diff
@@ -25,6 +25,9 @@
         </thead>
         <tbody>
             {% for file in files %}
+                {% if file.cover_image is not empty %}
+                {# Display files uploaded only on bulk upload mode #}
```

Comment:
A little nitpicky, but I think adding an indentation for the inner is a bit more readable.